.github/workflows/build.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       fail-fast: false
```

Comment:
1. `name: "PHP ${{ matrix.php }}, MySQL ${{ matrix.mysql }}"` -> There should be also info about Sylius version
2. There should be Sylius version and Symfony version in the matrix. Keep in mind we support both 1.9 and 1.10
3. We can bump node to 14.x
4. When you finalize the point 2 remember to add restricting Sylius and Symfony versions

As a reference you can use build from e.g. CMS plugin

Reply:
I can bump node version. Can I add 1 & 2 with Sylius 1.11 update?